optimade/models/structures.py

```diff
@@ -46,43 +46,45 @@ class StructureFeatures(Enum):
+    - `[ {"name": "Ti", "chemical_symbols": ["Ti", "vacancy"], "concentration": [0.9, 0.1]} ]`: any site with this species is occupied by a Ti atom with 90 % probability, and has a vacancy with 10 % probability.
+    - `[ {"name": "BaCa", "chemical_symbols": ["vacancy", "Ba", "Ca"], "concentration": [0.05, 0.45, 0.5], "mass": 88.5} ]`: any site with this species is occupied by a Ba atom with 45 % probability, a Ca atom with 50 % probability, and by a vacancy with 5 % probability. The mass of this site is (on average) 88.5 a.m.u.
+    - `[ {"name": "C12", "chemical_symbols": ["C"], "concentration": [1.0], "mass": 12.0} ]`: any site with this species is occupied by a carbon isotope with mass 12.
+    - `[ {"name": "C13", "chemical_symbols": ["C"], "concentration": [1.0], "mass": 13.0} ]`: any site with this species is occupied by a carbon isotope with mass 13.
+    - `[ {"name": "CH3", "chemical_symbols": ["C"], "concentration": [1.0], "attached": ["H"], "nattached": [3]} ]`: any site with this species is occupied by a methyl group, -CH3, which is represented without specifying precise positions of the hydrogen atoms.
 
     """
 
     name: str = Field(
         ...,
-        decsription="""REQUIRED; gives the name of the species; the **name** value MUST be unique in the :property:`species` list;""",
+        decsription="""Gives the name of the species; the **name** value MUST be unique in the `species` list.""",
     )
 
     chemical_symbols: List[str] = Field(
         ...,
-        description="""MUST be a list of strings of all chemical elements composing this species.
-
-- It MUST be one of the following:
+        description="""MUST be a list of strings of all chemical elements composing this species. Each item of the list MUST be one of the following:
 
-  - a valid chemical-element name, or
-  - the special value :val:`"X"` to represent a non-chemical element, or
-  - the special value :val:`"vacancy"` to represent that this site has a non-zero probability of having a vacancy (the respective probability is indicated in the :property:`concentration` list, see below).
+- a valid chemical-element name, or
+- the special value `"X"` to represent a non-chemical element, or
+- the special value `"vacancy"` to represent that this site has a non-zero probability of having a vacancy (the respective probability is indicated in the `concentration` list, see below).
 
--  If any one entry in the :property:`species` list has a :property:`chemical_symbols` list that is longer than 1 element, the correct flag MUST be set in the list :property:`structure_features` (see property `structure_features`_).""",
+If any one entry in the `species` list has a `chemical_symbols` list that is longer than 1 element, the correct flag MUST be set in the list `structure_features` (see property [structure_features](#structure_features)).""",
```

Comment:
Or does this internal tag linking work?
```suggestion
If any one entry in the `species` list has a `chemical_symbols` list that is longer than 1 element, the correct flag MUST be set in the list `structure_features` (see property structure_features).""",
```

Reply:
Internal linking doesn't work, again, thought I'd caught all of these...

Reply:
Ugh, there's a few left here for some reason. I'll fix them all in one go.